include/boost/math/tools/ulps_plot.hpp

```diff
@@ -247,11 +247,16 @@ class ulps_plot {
                 }
                 if (plot.clip_ > 0 && static_cast<PreciseReal>(abs(ulp[j])) > plot.clip_)
                 {
-                    continue;
+                   CoarseReal x = x_scale(plot.coarse_abscissas_[j]);
+                   PreciseReal y = y_scale(static_cast<PreciseReal>(ulp[j] < 0 ? -plot.clip_ : plot.clip_));
+                   fs << "<circle cx='" << x << "' cy='" << y << "' r='1' fill='" << color << "'/>\n";
```

Comment:
Should we maybe color the clipped values differently? Suppose red for all clips?

Reply:
One further point: If the condition number goes infinite somewhere, which is very common, the top of the plot is going to be a bloodbath, even if there's really nothing wrong with the implementation.

I'm not sure what to do about this, since it seems like something no one wants an option for!

Reply:
+1 on red for clipped values, and yes the top of the plot may be a bloodbath, but my feeling is it's better to know than not about this stuff.